deployments/gpu_plugin/overlays/nfd_labeled_nodes/add-nodeselector-intel-gpu.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-gpu-plugin
+spec:
+  template:
+    spec:
+      nodeSelector:
+        feature.node.kubernetes.io/pci-0300_8086.present: "true"
```

Comment:
nice. I had to go look up NFD and the PCI labelling :-). I presume this detects all our useable Intel GPUs? I wonder if it would also detect GPUs that are too old to run compute on though? I dunno...

Reply:
Good point. @uniemimu, do you know if NFD has anything more precise for discovering right nodes?

As long as this does not exclude nodes with gpu_plugin acceleration, it's better than nothing - which would deploy the plugin on all nodes.

Reply:
This is nice indeed. Just one concern from QAT perspective: the default label pattern (it's configurable) does not give the PCI IDs. We can only detect "Co-processor" (IIRC)...

Reply:
A QAT overlay for dpdk + c62x HW could be:

```
feature.node.kubernetes.io/pci-37c8_8086.present: "true"
feature.node.kubernetes.io/iommu-enabled: "true"
```
and `c6xxvf` for `KERNEL_VF_DRIVERS`.

Reply:
@grahamwhaley, GPU plugin itself will detect capabilities of the GPU on the host and expose those as node labels. (This is WIP by @uniemimu.) So `pci-0300_8086` is accurate enough before deploying the plugin.

Reply:
ack, sure, the plugin itself will ultimately decide. np.